cmd/der2ascii/main.go

```diff
@@ -70,7 +77,16 @@ func main() {
 			if pemBlock == nil {
 				break
 			}
-			inputs = append(inputs, input{comment: pemBlock.Type, bytes: pemBlock.Bytes})
+			if *pemPassword != "" {
+				bytes, err := x509.DecryptPEMBlock(pemBlock, []byte(*pemPassword))
```

Comment:
Apparently this is deprecated, but not in a way that's useful for this PR so okay whatever. :-)

Reply:
Yeah. It's deprecated because PEM encryption is bad (politifact check: true), but I think under go's stability policy it's not going anywhere.

Reply:
Yup. Yeah I was just waiting for CI to clear and then stepped away from screen. :-)